src/compiler.js

```diff
@@ -48,7 +64,24 @@ export class NexeCompiler {
     }
   }
 
-  get _deliverableLocation () {
+  _findPaddingSize (size, override) {
+    if (override === 0) {
+      return 0
+    }
+    size = override > size ? override : size
+    const padding = [3, 6, 9, 16, 25, 40].map(x => x * 1e6)
```

Comment:
They aren't defined anywhere but here I meant for it to be an implicit process where the best is just chosen. But it will need to shared to support the build process.